resources/favEvents/favEvents.route.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+
+const router = express.Router();
+
+const favEventsController = require('./favEvents.controller');
+
+router.get('/', async (req, res) => {
+  try {
+    const { u } = req.query;
```

Comment:
Please use a good naming convention, `u` to me is not intuitive

Reply:
Alright.